psydac/linalg/tests/test_kron_direct_solver.py

```diff
@@ -76,20 +73,29 @@ def matrix_to_sparse(A):
 def random_matrix(seed, space):
     A = StencilMatrix(space, space)
     p = space.pads[0]
+    dtype=space.dtype
 
     # for now, take matrices like this (as in the other tests)
+    if dtype==complex:
+        factor=1j
+    else:
+        factor=1
     A[:,-p:0   ] = 1
-    A[:, 0 :1   ] = (seed+10)*p
+    A[:, 0 :1   ] = (seed*factor+10)*p
     A[:, 1 :p+1] = -1
 
     return A
 
-#===============================================================================
-def random_vectordata(seed, npts):
+def random_vectordata(seed, npts, dtype):
+    #define our factor to choose our data type
+    if dtype==complex:
+        factor=1j
+    else:
+        factor=1
     # for now, take vectors like this (as in the other tests)
-    return np.fromfunction(lambda *point: sum([10**i*d+seed for i,d in enumerate(point)]), npts)
+    return np.fromfunction(lambda *point: sum([10**i*d+factor*seed for i,d in enumerate(point)]), npts,dtype=dtype)
```

Comment:
Please add an empty space after the comma (before `dtype=dtype`).

Reply:
[86b06e9](https://github.com/pyccel/psydac/pull/273/commits/86b06e97fda7a028f2d6ec3f23b6c81d44eaf696)